website/yw_root/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='index'),
+]
```

Comment:
Always end files with newline character 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline